### PR TITLE
Changes references of Nanotrasen to "The Company".

### PR DESCRIPTION
**Pull request by Kurgis**
Pre-approved changes checked off by ACCount. @drexample 
Merge when ready.

dadw

### CODE REVIEW

**Comment by Pariah919** [comment]
10/10

File: code/datums/setup_option/backgrounds/origin.dm [hunk 1]
```diff
@@ -28,8 +28,8 @@
 	desc = "The product of a widely held idea back on Earth that if one were to bring all of the Balkan populations on one colony ship, they would be forced to put aside their differences and work together to survive. \
 			Unfortunately, that idea failed miserably and is still failing. \
 			A jungle death world, full of hostile fauna and mired in endless armed conflict. \
-			The Serbian government occupied all of Predstraza�s space ports and controls most of the civilized regions. \
-			Located far from Hanza or NT territories, it�s a place favored for business by pirates, smugglers and all manner of outlaw."
+			The Serbian government occupied all of Predstraza�s space ports and controls most of the civilized regions. \
+			Located far from Hanza or NT territories, it�s a place favored for business by pirates, smugglers and all manner of outlaw."
```
Comment (mattroks101) [inline code/datums/setup_option/backgrounds/origin.dm]: Something weird seems to have happened to `'`.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 2]
```diff
@@ -150,11 +150,11 @@
...
 			One of the first colonies, because of how rich the planets are in resources, and their abundance - the system being composed of more than a hundred of large celestial bodies. \
-			It�s also an extremely valuable place for scientists, due to its habitable planets, the black hole and rare materials. \
+			It�s also an extremely valuable place for scientists, due to its habitable planets, the black hole and rare materials. \
 			End Point was never controlled by a single power. \
 			Smaller colony ships, belonging to third-world countries, damaged ships or just exploration cruisers - they have all found their place here, guided by a black hole and the riches highlighted by it. \
 			Even before the war it was full of conflicts between local governments, pirates and corporations, and it just got worse afterwards. \
-			While it�s formally under Hanza control now, the war resulted in a fall of many governments, thus the anarchy spreads, and patchwork states are being born and die every year. \
+			While it�s formally under Hanza control now, the war resulted in a fall of many governments, thus the anarchy spreads, and patchwork states are being born and die every year. \
```
Comment (mattroks101) [inline code/datums/setup_option/backgrounds/origin.dm]: Here as well.

File: code/game/objects/items/weapons/clown_items.dm [hunk 3]
```diff
@@ -92,7 +92,7 @@
 		return
 	..()
 
-/obj/item/weapon/soap/nanotrasen
+/obj/item/weapon/soap/company
 	desc = "A NeoTheology-brand bar of soap. Smells of biomatter."
```
Comment (mattroks101) [inline code/game/objects/items/weapons/clown_items.dm]: This isn't nanotrasen or company soap, it's NeoTheo soap.

**Comment by ACCount12** [comment]
nice meme, real shame to crush this dream

**Pull request closed without merging**